src/partials/Banner.svelte

```diff
@@ -0,0 +1,100 @@
+<script>
+  export let titleText = "Page Title";
+  export let subtitleText = "";
+  export let bgColor = "";
+  export let bannerImg = "";
+  export let bannerImgMobile = "";
+  export let overlayColor = "var(--gray-100)";
+  export let overlayOpacity = 0.6;
+  export let titleFontSize = "";
+  export let titleFontWeight = "";
+
+  const getBackgroundImageValue = (str) =>
+    /\//.test(str) ? `url(${str})` : str;
+  $: bgImgLgValue = getBackgroundImageValue(bannerImg);
+  $: bgImgSmValue = getBackgroundImageValue(bannerImgMobile);
+</script>
+
+<style>
+  .banner {
+    min-height: 16rem;
+    color: var(--white);
+    background-image: var(--banner-img, none);
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-color: var(--banner-bg-color, var(--gray-80));
+    padding: var(--spacing-48) 0;
+  }
+
+  @media (max-width: 672px) {
+    .banner {
+      min-height: 14rem;
+      background-image: var(--banner-img-mobile, var(--banner-img), none);
+    }
+  }
+
+  .banner-content > * + * {
+    margin-top: var(--banner-child-v-spacing, 2.5rem);
```

Comment:
This variable isn't set anywhere so it will always be `undefined` and default to 2.5rem.